R/repositories.R

```diff
@@ -21,18 +29,19 @@ BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
     if (length(conflicts)) {
         txt <- paste(
             "'getOption(\"repos\")' replaces Bioconductor standard ",
-            "repositories, see '?repositories' for details"
+            "repositories, see ",
+            "'help(\"repositories\", package = \"BiocManager\")' for details."
         )
         fmt <- paste0(
             .msg(txt, exdent = 0),
-            "\n\nreplacement repositories:",
+            "\n\nReplacement repositories:",
```

Comment:
I find the extra newline here distracting

Reply:
@mtmorgan The difference is the capital `R` in 'replacement'

Reply:
I understand, but since you were revising I thought I'd mention...

Reply:
Ah, did you mean the extra `\n`? Yes, I can remove it.